Assets/Scripts/Gameplay/GameplayObjects/Character/LifeState.cs

```diff
@@ -0,0 +1,12 @@
+using System;
```

Comment:
In this branch, I see both a "LifeState.cs" file and a "NetworkLifeState.cs" file.
It confused me at first, I thought there was a duplicate. Do we have naming conventions for enums? Should an enum really live on its own in a single file?

Reply:
moved to the NetworkLifeState